internal/cfgplugins/qos.go

```diff
@@ -372,3 +375,53 @@ func GetPolicyCLICounters(t *testing.T, dut *ondatra.DUTDevice, policyName strin
 		return ""
 	}
 }
+
+func ConfigureQosDscpRemarkSpecific(t *testing.T, dut *ondatra.DUTDevice) {
+	if deviations.QosRemarkOCUnsupported(dut) {
+		switch dut.Vendor() {
+		case ondatra.ARISTA:
+			aristaConfigureRemarkIpv4(t, dut)
+			aristaConfigureRemarkIpv6(t, dut)
+		}
+	}
```

Comment:
Please include the OC standard way to rewrite/remark DSCP.  The key path is:

[/qos/classifiers/classifier/terms/term/actions/remark/config/set-dscp](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-qos.html#qos-classifiers-classifier-terms-term-actions-remark-config-set-dscp)

Reply:
it is included in qos.go file in line number 156.
		if !deviations.QosRemarkOCUnsupported(dut) {
			action.GetOrCreateRemark().SetDscp = ygot.Uint8(class.RemarkDscp)
		}

since it is aborting the script, we are adding it as another function.

Reply:
ok, please add comments explaining this ordering of the configuration both inline where ConfigureQosDscpRemarkSpecific is called, and as a description for the function.

Also this should be a private function, so it should be lower case.